vipps-partner-api.md

```diff
@@ -260,23 +263,87 @@ We are working on simplifying this in the future.
 This may be useful:
 [Typical reasons for delays](https://github.com/vippsas/vipps-partner/blob/main/README.md#typical-reasons-for-delays).
 
-### Scenario where the merchant does not have a Merchant Agreement with Vipps
+### Scenarios
 
-In order to get approved to have a product (i.e eCom or Recurring payments) with Vipps the merchant must first have a formal agreement with Vipps. Referred to as a Merchant Agreement. This is the current flow for the Merchant with or without a Merchant agreement when Prefill is used. We are actively looking to improve the user experience in the case where Prefill is used without a Merchant Agreement.
+In order to get approved to have a product ("Vipps på nett", "Vipps Login", etc)
+with Vipps the merchant must first have a formal agreement with Vipps, called
+the Merchant Agreement (MA).
 
-The state flow can be described as follows.
+Merchants need both a Merchant Agreement (MA) and a Product Order (PO) to
+be able to accept Vipps payments:
+* MA: An agreement between the merchant and Vipps, signed with BankID.
+  The MA contains information about all direct and indirect owners, any
+  politically exposed persons, etc. The MA is signed with BankID.
+* PO: This is an order for "Vipps på nett", "Vipps Login", etc. The merchant
+  must provide some information about thew use, whether the3 cardholder is
+  present, etc. The PO is not signed with BankID.
 
-![Merchant agreement flow](images/merchant-agreement-flow.png)
+A merchant may order a Vipps product (submit a PO) with or without a Merchant Agreement (MA).
 
-The merchant will now see the prefilled product order waiting for them in the portal under "Produktbestillinger klare til innsendelse" in their home screen. (see highlighted area in image)
-
-![example-portal-with-preorder](images/example-portal-with-preorder-waiting.png)
-
-Alternatively the URL created in the initial prefill can be reused for up to 15 days.
+**Please note:** The only method Vipps has to verify that a user has the right
+to sign a MA for a merchant is by using data from
+[Brønnøysundregistrene](https://brreg.no).
+It is therefore a requirement that the user logging in on
+[portal.vipps.no](https://portal.vipps.no)
+is registered as chairman of the board ("styreleder") or CEO ("daglig leder").
+
```

Comment:
Probably add some information that the link is not needed, that with a valid MA, the merchant can at any time find the prefill on the MP home page.

Reply:
Takk, lagt til.